src/main/java/com/salesforce/dynamodbv2/mt/util/CompositeStrings.java

```diff
@@ -64,7 +64,7 @@ public String join(Iterable<String> values) {
      * @return Sequence of keys contained in the given composite key
      */
     public Iterator<String> split(String composite) {
-        return new Iterator<>() {
+        return new Iterator<String>() {
```

Comment:
Is mt-dynamo going to stay in jdk11 mode? Are we running the risk that we accidentally introduce issues like this again without catching it as part of the mt-dynamo build?